common/amk/amk.c

```diff
@@ -0,0 +1,429 @@
+#include "amk.h"
+#include "source/main_module/can/can_parse.h"
+
+/* NOTE:
+ * Step 1 is turning on LV
+ * Step 3 happens when HV is turned on and precharging starts (I think this is actually step 2)
+ *  I can check when this is done with precharge complete and then move 
+ *  onto other steps
+ *  MAYBE we can check when AMK_bSystemReady is on and display a message
+ *  on LCD so they know when to turn on the HV to start the precharging.
+ *  Not sure if I need that
+ */
+
+static void turnMotorOn(amk_motor_t* motor);
+static void motorRunning(amk_motor_t* motor);
+static void turnMotorOff(amk_motor_t* motor);
+static void motorGetData(amk_motor_t* motor);
+static void motorReset(amk_motor_t* motor);
```

Comment:
Sorry this is super nitpicky lmao
```suggestion
static void resetMotor(amk_motor_t* motor);
```